File: wear/src/main/java/net/osdn/gokigen/joggingtimer/stopwatch/IDatabaseReloadCallback.java
package net.osdn.gokigen.joggingtimer.stopwatch;

import java.util.ArrayList;

/**
 *
 *
 */
public interface IDatabaseReloadCallback
{
    void dataIsReloaded(ArrayList<Long> list);
    void referenceDataIsReloaded(int id, ArrayList<Long> list);
}

File: wear/src/main/java/net/osdn/gokigen/joggingtimer/stopwatch/IDatabaseReloadCallback.kt
package net.osdn.gokigen.joggingtimer.stopwatch

import net.osdn.gokigen.joggingtimer.stopwatch.timer.LapTimeInfo

/**
 *
 *
 */
interface IDatabaseReloadCallback
{
    fun dataIsReloaded(timelist: ArrayList<LapTimeInfo>)
    fun referenceDataIsReloaded(id: Int, timelist: ArrayList<Long>)
}
